sagemaker/sagemaker.tfaws.w

```diff
@@ -0,0 +1,32 @@
+bring "./types.w" as types;
+bring aws;
+
+pub class SageMaker_tfaws impl types.ISageMaker {
+  pub endpointName: str;
+  pub inferenceComponentName: str;
+
+  new(endpointName: str, inferenceComponentName: str) {
+    this.endpointName = endpointName;
+    this.inferenceComponentName = inferenceComponentName;
+  }
+
+  pub inflight invoke(body: Json, options: types.invocationOptions?): types.invocationOutput{
+    return SageMaker_tfaws._invoke(this.endpointName, body, this.inferenceComponentName, options);
+  }
+
+  pub onLift(host: std.IInflightHost, ops: Array<str>) {
+    if let lambda = aws.Function.from(host) {
+      lambda.addPolicyStatements({
+        actions: ["sagemaker:InvokeEndpoint"],
+        effect: aws.Effect.ALLOW,
+        resources: [
+          "arn:aws:sagemaker:*:*:endpoint/{this.endpointName.lowercase()}",
+          "arn:aws:sagemaker:*:*:inference-component/{this.inferenceComponentName.lowercase()}"
+        ]
+      });
+    }
```

Comment:
Throw an exception if this is the wrong target

Reply:
I think @ekeren wants it to be able to be used from the sim target too